### PR TITLE
Fixes #17183: Add Create Node endpoint in API documentation

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17183

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

File: webapp/sources/api-doc/components/schemas/node-add.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+type: array
+items:
+  type: object
+  required:
+    - id
+    - hostname
+    - status
+    - os
+  #  - policyServerId [Optional, by default "root"]
```
Comment (amousset) [inline webapp/sources/api-doc/components/schemas/node-add.yml:11]: why is it commented?

Comment (ElaadF) [reply webapp/sources/api-doc/components/schemas/node-add.yml:11]: I thought these were required fields, but there are optional so I comment them for better understanding, to keep track

**Pull request merged**